frontend/public/components/deployment-config.tsx

```diff
@@ -99,6 +99,8 @@ export const DeploymentConfigDetailsList = ({dc}) => {
     <dd>{dc.spec.minReadySeconds ? pluralize(dc.spec.minReadySeconds, 'second') : 'Not Configured'}</dd>
     {triggers && <dt>Triggers</dt>}
     {triggers && <dd>{triggers}</dd>}
+    <dt>Tolerations</dt>
+    <dd><a className="co-m-modal-link" onClick={Kebab.factory.ModifyToleration(DeploymentConfigModel, dc).callback}>{pluralize(_.size(dc.spec.template.spec.tolerations), 'Toleration')}</a></dd>
```

Comment:
I think it would make sense to add this to `ResourceSummary` as an option (off by default). Similar to node selector, annotations, etc.

https://github.com/spadgett/console/blob/e884e44bdf55005adba898f50f73e5bf95493e4f/frontend/public/components/utils/details-page.tsx#L38-L39

We might need to add a `tolerationsPath` prop that defaults to `spec.template.spec.tolerations`. The path could be passed through to the dialog.

Reply:
I don't think I need to, I can get it to work without doing that. Plus I think trying to pass a new prop to the modal would involve changing the KebabAction type and would have larger ramifications.